docs/internals/contributing/triaging-tickets.txt

```diff
@@ -82,7 +82,7 @@ By way of example, here we see the lifecycle of an average ticket:
 * Alice updates the pull request, fixing the implementation. She removes the
   "patch needs improvement" flag.
 
-* Daisy reviews the pull request and marks the ticket as "Ready for checkin".
+* Daisy reviews the pull request and marks the ticket as "Ready for check-in".
```

Comment:
I'm not certain if it's worth changing these if it's going to become out of sync with Trac, where it's called "checkin".